app/assets/stylesheets/pages/_homepage.scss

```diff
@@ -30,6 +30,6 @@
   cursor: pointer;
 }
 
-#gh-logo {
+#turing-logo {
```

Comment:
We do use the class `turing-logo` for the header; perhaps it would be better to reclassify that one to an id as well & have one be `turing-logo-lg` & one `turing-logo-sm`?

Reply:
Works for me.

Reply:
Updated in 24585bc to use `#turing-logo-sm`. The top left nav uses `.turing-logo`, which appears to not be defined anywhere so I left things where they were. I assume we can circle back later to fix if we want to. Let me know if I missed something.